server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -411,7 +525,8 @@ public void testAckListenerReceivesNacksFromFollowerInHigherTerm() {
     }
 
     public void testSettingInitialConfigurationTriggersElection() {
-        final Cluster cluster = new Cluster(randomIntBetween(1, 5));
+        final int nodeCount = randomIntBetween(1, 5);
+        final Cluster cluster = new Cluster(nodeCount);
```

Comment:
maybe add a `size()` method to `cluster` instead of manually managing `nodeCount` here

Reply:
I added this in a6f50e3f4a00ed87b600ef325af463c3fbafa3c6.